execution_engine_testing/tests/src/test/storage_costs.rs

```diff
@@ -154,10 +75,10 @@ fn initialize_isolated_storage_costs() -> InMemoryWasmTestBuilder {
         .with_current_protocol_version(*DEFAULT_PROTOCOL_VERSION)
         .with_new_protocol_version(*NEW_PROTOCOL_VERSION)
         .with_activation_point(DEFAULT_ACTIVATION_POINT)
-        .with_new_wasm_config(*STORAGE_COSTS_ONLY)
```

Comment:
You need to use `STORAGE_COSTS_ONLY` in the upgrade request below rather than carry on existing config

Reply:
The test now uses `STORAGE_COSTS_ONLY`